feature-flags.json

```diff
@@ -4,5 +4,6 @@
 	"ideaHubModule",
 	"swgModule",
 	"userInput",
-	"zeroDataStates"
+	"zeroDataStates",
+	"ga4ActivationBanner"
```

Comment:
In the future let's keep these alphabetically organised, it's minor but just makes them a bit easier to scan 🙂

Reply:
Thanks, @tofumatt